test/unit/rules/no-redundant-landmark-role-test.js

```diff
@@ -7,9 +7,46 @@ generateRuleTests({
 
   good: [
```

Comment:
Can you add a commen to indicate which of these test cases is checking the behavior when `checkAllElementRoles` is not specified?